conference/LinuxPlumbers2021/xdp-multi-buff.org

```diff
@@ -198,6 +198,10 @@ BPF_CALL_2(bpf_xdp_adjust_data, struct xdp_buff *, xdp, u32, offset)
 }
 #+end_src
 
+** Slide: XDP multi-buffers: new eBPF helpers (4/4)                  :export:
+- /bpf_xdp_output/ and /bpf_perf_event_output/:
+  - Helpers are updated and can copy the content of all buffers
+
```

Comment:
@chaudron maybe you can add some code snippet here, what do you think?

Reply:
Code snippet on how to use the helper? That I can do, but adding the changes to the code makes no sense as it's not well contained.

Reply:
@chaudron what I mean is try to summarize what you changed in the upstream code, I think you can take a look to the presentation to say what I mean. Btw to add code snippet you just need the #+begin_src C and #+end_src C tags